api/providers/views.py

```diff
@@ -611,11 +643,90 @@ class RegistrationProviderSchemaList(JSONAPIBaseView, generics.ListAPIView, List
 
     serializer_class = RegistrationSchemaSerializer
 
-    def get_provider(self):
-        return RegistrationProvider.objects.get(_id=self.kwargs['provider_id'])
-
     def get_default_queryset(self):
         return self.get_provider().schemas.get_latest_versions(request=self.request, invisible=True).filter(active=True)
 
     def get_queryset(self):
         return self.get_queryset_from_request()
+
+
+class RegistrationProviderRegistrationList(JSONAPIBaseView, generics.ListAPIView, ListFilterMixin, ProviderMixin):
+    """The documentation for this endpoint can be found [here](https://developer.osf.io/#operation/preprint_providers_preprints_list).
+    """
+    provider_class = RegistrationProvider
+    permission_classes = (
+        drf_permissions.IsAuthenticated,
+        base_permissions.TokenHasScope,
+        MustBeModerator,
+    )
+
+    ordering = ('-created')
+
+    serializer_class = DraftRegistrationSerializer
+
+    required_read_scopes = [CoreScopes.NODE_REGISTRATIONS_READ]
+    required_write_scopes = [CoreScopes.NULL]
+
+    view_category = 'registration-providers'
+    view_name = 'registrations-list'
+
+    def get_default_queryset(self):
+        return DraftRegistration.objects.filter(
+            provider=self.get_provider(),
+            registered_node__registration_approval__state=RegistrationApproval.APPROVED,
+        )
```

Comment:
Shouldn't this actually be resolving to the Registration instead, though? I thought we had a discussion about that when we were deciding whether we could attach this to the AbstractNode or not, and that we'd have to take the performance hit and resolve these to the Registrations. 

I think it's going to confuse the ember app if we have it be the draft registration we serialize here, but actually being a registration.